cypress/locators/show-edit-pod/index.js

```diff
@@ -1,22 +1,32 @@
 import {
-  SHOW_EDIT_POD_TITLE,
-  SHOW_EDIT_SAVE_BUTTON,
-  SHOW_EDIT_CANCEL_BUTTON,
-  SHOW_EDIT_DELETE_BUTTON,
-  SHOW_EDIT_POD_EDIT,
-  SHOW_EDIT_POD_CONTENT_BOX,
+  POD,
+  CONTENT_FORM_SHOW_EDIT_POD,
```

Comment:
Why did you switch around the format of these locators from, for example, SHOW_EDIT_POD_SAVE_BUTTON to SAVE_BUTTON_SHOW_EDIT_POD when all other components are the original format of COMPONENT_ELEMENT as far as possible? Similarly for the corresponding indexes from showEditSaveButton to saveButtonShowEditPod? The format should be as uniform as possible.

Reply:
I have changed the order now.